week6/04-grazhdanskaia-oborona/task.py

```diff
@@ -0,0 +1,49 @@
+def main():
+    n = int(input())
+    village_list = list(input().split())
+    m = int(input())
+    bomb_list = list(input().split())
+    for i in range(n):
+        pos = village_list[i]
+        village_list[i] = (int(pos), i + 1)
```

Comment:
provide class to describe meaning of 1st and 2nd value in tuple (with the help of filed names)